tests/resources/technical-records.json

```diff
@@ -27985,5 +27985,170 @@
       }
     ],
     "vin": "DP76UMK4DQLTOT400066"
+  },
+  {
+    "partialVin": "111222",
```

Comment:
This partialVin already exists in the data, this could potentially cause issues when the test pack runs.

Reply:
There will be a few changes before approval is required, this was quickly pushed up so it could be tested. Will let you know when code is ready for PR.